recipes/liburing/all/patches/disable_samples_and_tests.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/Makefile b/Makefile
+index 73d021c..08ebd96 100644
+--- a/Makefile
++++ b/Makefile
+@@ -7,9 +7,9 @@ INSTALL=install
+ default: all
+ 
+ all:
+-	@$(MAKE) -C src
+-	@$(MAKE) -C test
+-	@$(MAKE) -C examples
++	@$(MAKE) -C src liburing.a
```

Comment:
Why does this unconditionally build a static liburing, is it used to make the shared version if applicable? I would expect `make src` to do the correct thing.

Reply:
This was a mistake from my part.
 liburing Makefile builds unconditionnally both static and shared library. The recipe then cleanup in the package phase